src/styles/abstracts/_variables.scss

```
@@ -6,6 +6,9 @@ $light: #fff;
 $dark: #000;
 $shuttle-gray: #5a6570;
 $picton-blue: #47a1f5;
+$royal-blue: #5358d8;
+$cornflower-blue: #638cff;
+$signup-button-color: rgba(0, 0, 0, 0.16);
```

Isn't it the shadow of a button?

```suggestion
$signup-button-shadow: rgba(0, 0, 0, 0.16);
```

---

Yes, I will change it immediately. Sorry for my heedlessly work